modules/oe_editorial_corporate_workflow/config/install/workflows.workflow.oe_corporate_workflow.yml

```diff
@@ -63,23 +63,17 @@ type_settings:
       from:
         - needs_review
       to: draft
-      weight: -6
-    needs_review_to_needs_review:
-      label: 'Needs Review to Needs Review'
-      from:
-        - needs_review
-      to: needs_review
-      weight: -7
+      weight: -5
     needs_review_to_request_validation:
       label: 'Needs Review to Request Validation'
       from:
         - needs_review
       to: request_validation
-      weight: -5
+      weight: -4
     publish:
```

Comment:
All the transitions are using the pattern `<from_state>_to_<to_state>` except this one. Let's take the chance to normalise this maybe?